plugins/carbon_black_cloud/icon_carbon_black_cloud/tasks/monitor_alerts/task.py

```diff
@@ -241,7 +241,7 @@ def get_observations(
                     has_more_pages = True
 
             # we use if not observations, vs else, to ensure we pick up on the case where we dedupe every observation
-            if not observations:
+            if not observations and not job_time_exceeded:
```

Comment:
Can we just add a comment around this logic so we know looking back why we have the `and not job_time_exceeded`

Reply:
I have added a comment around this